browser/ui/webui/new_tab_page/branded_wallpaper.h

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) 2019 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// you can obtain one at http://mozilla.org/MPL/2.0/.
+
+#ifndef BRAVE_BROWSER_UI_WEBUI_NEW_TAB_PAGE_BRANDED_WALLPAPER_H_
+#define BRAVE_BROWSER_UI_WEBUI_NEW_TAB_PAGE_BRANDED_WALLPAPER_H_
+
+#include <memory>
+#include <string>
+#include <vector>
+
+class BrandedWallpaperLogo {
```

Comment:
I had `struct` at first, but compiler kept giving me errors about complex types. I'll try again, since I now think this can all be one simple data structure without the `BrandedWallpaper.BrandedWalllpaperLogo` child property. This is because the data structure here can be a different structure compared to the one received from remote. Originally I was thinking one large data structure with multiple `Logo` properties for each region, but we're now decoupled from the remote data source.

Reply:
This is not applicable anymore